lib/sorbet-rails/active_record_rbi_formatter.rb

```diff
@@ -85,11 +85,16 @@ def generate_active_record_relation_rbi
       # parameters than the ones defined by `create_elem_specific_query_methods` so
       # we need to match the signatures in that conflicting rbi.
       build_methods = %w(new build create create!)
+      if Rails.version =~ /^5\.\d/
```

Comment:
We can probably change this to use Gem::Version comparison so we can check if it's >= 6.0.3?